src/bin/tupm/ui.rs

```diff
@@ -691,8 +691,8 @@ impl Ui {
 
         let ui_tx_clone = ui.ui_tx.clone();
         account_list.set_on_select(move |s, account| {
-            let mut detail = match s.find_id::<TextView>(VIEW_ID_DETAIL) {
-                Some(x) => x,
+            let mut detail: ViewRef<TextView> = match s.find_id(VIEW_ID_DETAIL) {
+                Some(acc_det) => acc_det,
```

Comment:
You could also write this:

```rust
s.call_on_id(VIEW_ID_DETAIL, |detail: &mut TextView| {
    detail.set_content(render_account_text(account, false));
    ui_tx_clone.send(UiMessage::UpdateStatus).unwrap();
});
```

Reply:
Right! Thank you for the review! The result certainly is very elegant. I included it into the PR and pushed the result (after testing ofcourse).

Reply:
P.S. Not sure about the button "Resolve Conversation" below. For whom is that meant?